slidez-core/src/main/java/com/binarystudio/academy/slidez/infrastructure/security/SecurityConfig.java

```diff
@@ -11,12 +10,10 @@
 @EnableWebSecurity
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
+	private final String[] PUBLIC_URLS = { "/auth/**", "/swagger-ui/**", "/api-docs/**" };
```

Comment:
please, add "/" and "/health" to public_urls

Reply:
Done!